source/classes.tex

```diff
@@ -6660,53 +6667,142 @@
 A defaulted comparison operator function\iref{over.binary}
 for some class \tcode{C}
 shall be a non-template function
-declared in the \grammarterm{member-specification} of \tcode{C}
+defined in the \grammarterm{member-specification} of \tcode{C}
 that is
 \begin{itemize}
 \item
-a non-static const member of \tcode{C} having
-one parameter of type \tcode{const C\&}, or
+a non-static const non-volatile member of \tcode{C} having
+one parameter of type \tcode{const C\&} and either
+no \grammarterm{ref-qualifier} or
+the \grammarterm{ref-qualifier} \tcode{\&}, or
 
 \item
 a friend of \tcode{C} having either
 two parameters of type \tcode{const C\&} or
 two parameters of type \tcode{C}.
 \end{itemize}
-
-\pnum
-A defaulted comparison operator function for class \tcode{C}
+A defaulted comparison operator function for class \tcode{C} that
+is not defined as deleted is
+\indextext{operator!comparison!defaulted!implicitly defined}%
+\defn{implicitly defined}
+when it is odr-used or needed for constant evaluation.
+Name lookups in the defaulted definition
+are performed from a context equivalent to
+the \grammarterm{function-body} of the defaulted \tcode{operator@} function.
+A defaulted comparison operator function shall be
+defaulted on its first declaration.
+
+\pnum
+A defaulted \tcode{<=>} or \tcode{==} operator function for class \tcode{C}
+\indextext{operator!\idxcode{<=>}!deleted}%
+\indextext{operator!\idxcode{==}!deleted}%
 is defined as deleted if
 any non-static data member of \tcode{C} is of reference type or
-\tcode{C} is a union-like class\iref{class.union.anon}.
+\tcode{C} has variant members\iref{class.union.anon}.
+
+\pnum
+A binary operator expression \tcode{a @ b} is
+\indextext{operator!\idxcode{@}!usable}%
```

Comment:
drop the \idxcode{@}; it doesn't help in the index entry.

Reply:
This is how we format operators in the index, e.g.:
```
\indextext{operator!\idxcode{+=}}%
\indextext{operator!\idxcode{<=>}!deleted}%
\indextext{\idxcode{++}|see{operator, increment}}%
```